ui/main/src/app/business/templateGateway/cardTemplateGateway.ts

```
@@ -60,12 +63,22 @@ export class CardTemplateGateway {
             logger.info(` Template : no getUserResponse method provided , valid set to false`);
             return {valid: false, errorMsg: 'Impossible to respond due to a technical error in the template'};
         };
+
+        CardTemplateGateway._functionToEditCard = () => {};
+    }
+
+    public static registerFunctionToEditCard(editCard: Function) {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)